src/_utils/offline/service-worker/sw.template.js

```diff
@@ -0,0 +1,57 @@
+const cacheName = "<%= cacheName %>"
+
+/* eslint-disable quotes */
+const cacheList = JSON.parse('<%= files %>')
```

Comment:
Actually, we don't need JSON.parse here. It can just be like this 

`const cacheList = <%= files %>`

But this will break babel and eslint so I choose this approach